.github/workflows/Tests.yml

```diff
@@ -3,7 +3,6 @@ name: Tests
 on:
   push:
     branches:
-      - master
```

Comment:
Why no more testing on the release branch? I think it's still a good idea to show that it's passing there (to ensure everything got to the master branch/release tag correctly)

Reply:
Oh yeah, no I'm just creating two separate workflows, because if it fails on develop, it shows the badge as failing on master 😆